File: Python/PascalsTriangle.py
def pascalsTriangle(numRows):
    result = []
    first = [1]
    result.append(first)

    for i in range(1,numRows):
        prevList = [0] + result[i-1] + [0]
        tempList = []
        for j in range(len(prevList)-1):
            tempList.append(prevList[j]+prevList[j+1])
        result.append(tempList)

    return tempList

File: Javascript/PascalsTriangle.js
var pascalsTriangle = function(numRows){
    let first = [1]
    let result = []
    result.push(first)

    for(let i =1;i<numRows;i++){
        let tempList = []
        let prevList = [0].concat(result[i-1],[0])
        for(let j=0;j<prevList.length-1;j++){
            tempList.push(prevList[j]+prevList[j+1])
        }
        result.push(tempList)
    }
    return tempList
}